example/models_directory/st/det/ml.json

```diff
@@ -1,6 +1,6 @@
 {
     "method": {
-        "...": "include(../../../methods_directory/regression.json)"
+        "...": "include(../../../method.json)"
```

Comment:
1. I guess this is temporary until we define the pseudopotential method? If yes, we should put a todo, perhaps a similar one to the `"temporarily": "deactivated"` used above.
2. The use of `"temporarily": "deactivated"` could also be explained in the README so we don't forget and come back to clean it.

Reply:
Ok, good idea. Perhaps `"@todo": "temporarily deactivated"` would be a good solution. I will label the temporary changes accordingly.